locales/en.json

```diff
@@ -382,6 +382,7 @@
   "UploadForm.button.confirm": "Confirm",
   "UploadForm.button.mintUrl": "Has a URL, Arweave or IPFS hash? Mint NFT here",
   "UploadForm.button.selectFile": "Select a file",
+  "UploadForm.button.size.limit": "(Maximum upload file size: 20 MB)",
```

Comment:
```suggestion
  "UploadForm.button.size.limit": "(Recommended file size: < 20 MB)",
```

Reply:
[💬 Update limit text to recommend file size](https://github.com/likecoin/app-like-co/pull/498/commits/b971b5dc6541b8b6ff6c407e197668faa079d8a0)